clang/test/CodeGenSYCL/address-space-deduction.cpp

```diff
@@ -0,0 +1,74 @@
+// RUN: %clang_cc1 -triple spir64 -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
```

Comment:
Please add a comment here about what this test tests.

Reply:
Done.